src/platform/windows.rs

```diff
@@ -51,6 +51,20 @@ pub enum BackdropType {
     TabbedWindow = 4,
 }
 
+/// Describes if a window is activated by the mouse.
+///
+/// For a detail explanation, see[`WM_MOUSEACTIVATE docs`].
+///
+/// [`WM_MOUSEACTIVATE docs`]: https://learn.microsoft.com/en-us/windows/win32/inputdev/wm-mouseactivate
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Hash)]
```

Comment:
This enum should be non-exhaustive.

Reply:
None of our public enums are non-exhaustive, why should we make an exception for this one?